specs/examples/README.md

```diff
@@ -0,0 +1,89 @@
+- CDN: Vanilla JS and CSS
+    - Uses the CDN builds of packages via unpkg.com
+    - Known issue: fonts are not included in the [existing example](https://github.com/ni/nimble/tree/0fe8cc54f7bd444830e3974969519593b532b083/packages/nimble-components#prototyping-in-a-static-webpage)
+        - We should move all the pre-built cdn bundle stuff to a [separate package](https://twitter.com/justinfagnani/status/1521953590931189760?s=20&t=zejlKVWCbEBce8zYFO7Uyg), ie:
+            - `@ni/nimble-bundle`
+                - alternative names:
+                    - `@ni/nimble-components-cdn`
+                    - `@ni/nimbl-components-bundle`
+                    - `@ni/nimble-cdn-bundle`
+                    - `@ni/nimble-cdn`
+        - Alternative:
+            - We could add a build of our CSS + font assets to the existing nimble-components bundle build which will grow the package size and pose the risk of none cdn users accidentally leveraging the contents (as described in the separate package thread linked above).
+- Angular
+- Blazor
+
+### File structure
+  - examples/
+    - `<framework>`/
+        - README.md: Table of contents for the examples
+        - getting-started/
+            - README.md: Should include a Stackblitz link to open the example immediately in addition to other content
+            - package.json
+            - source/
+        - `<other examples>`: TBD
+
+Notes:
+- A subfolder per `<framework>`, i.e.: `vanilla`, `cdn`, `angular`, `blazor`
+- Each `<framework>` will have a `getting-started` example.
+    - Getting started example will be linked from each library package README
+    - Getting started examples assumes brand new developers who may not have web development experience:
+        - guidance on IDEs
+        - step-by-step instructions
+        - links to learn more about general web topics they need to be aware of (CSS custon properties, shadow slots, etc.)
+- Each example participates in the monorepo build
+- Each example follows Web Chapter conventions:
+    - Linting configured
+    - Testing configured
+    - Detailed README
+- Issue templates should be updated to include the Stackblitz urls to make it easier to report issues
+
+### Exisiting examples
+- The current [angular client example](https://github.com/ni/nimble/tree/main/angular-workspace/projects/example-client-app) and [blazor client example](https://github.com/ni/nimble/tree/main/packages/nimble-blazor/Examples)
+    - Will stay in current locations (angular-workspace and blazor workspace) as they are primarily developer oriented tools and not user facing examples
+    - Their READMEs should direct end users to the `<framework>` examples folders
+    - They will be renamed to `kitchen-sink` examples
+    - Continue to not have a Vanilla kitchen sink example
+    - Alternatives:
+        - Move the kitchen-sink examples to the examples/ folder
```

Comment:
After [discussion](https://github.com/ni/nimble/pull/1107#discussion_r1130170961) in a PR changing the title of these examples I vote for a name like `all-components` rather than `kitchen-sink`. It's not a showstopper, but idioms like "kitchen sink" are a bit less approachable for people whose first language language isn't English (not all languages use that idiom).

```suggestion
    - They will be renamed to `all-components` examples
    - Continue to not have a Vanilla all components example
    - Alternatives:
        - Move the all-components examples to the examples/ folder
```